src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/BurdenEstimateTests.kt

```diff
@@ -111,6 +143,17 @@ class BurdenEstimateTests : DatabaseTest()
         db.addResponsibility(setId, touchstoneId, scenarioId)
     }
 
+    private fun setUpWithSet(db: JooqContext): Int
```

Comment:
Mostly duplicating `setUp`? Also, make it could be named more clearly as something like `setUpWithBurdenEstimateSet`? I was reading `set` as a verb initially.